src/config/dynamic-theme-fixes.config

```
@@ -1402,6 +1402,13 @@ CSS
 
 ================================
 
+azurlane.koumakan.jp
+
+INVERT
```

```suggestion
IGNORE IMAGE ANALYSIS
```
That should work a little bit better.